packages/x-tree-view/typing.d.ts

```diff
@@ -0,0 +1,2 @@
+// After pnpm migration the dynamically imported types have to be defined explicitly
+/// <reference types="@emotion/styled" />
```

Comment:
Would you instead prefer to avoid creating a separate file, but add this sort of line in the `include` list in `tsconfig.build.json`?
```
"./node_modules/@emotion/styled/types/index.d.ts"
```

Reply:
IMO yes, because it will be more centralised, and this notation with commented references is special

Reply:
I've refactored all `tsconfig.json` and `tsconfig.build.json` files to:
1. Avoid including any files that are not within `src`
2. Move needed types into `compilerOptions.types`
3. Remove not needed explicit types
4. Move every relevant previously defined entry from `typings` into `compilerOptions.types`

Feel free to check the latest config. 😉 
If you have any additional feedback regarding it—please reopen the discussion. 👍